web/templates/web/garden/home.html

```diff
@@ -46,16 +47,16 @@
         <div class="col-12 col-lg-10"></div>
         <h3 class="{% subheading_classes %}">Click the links below to learn more about Project GARDEN!</h3>
         <p>
-            <a href="#">What is Project GARDEN?</a>
+            <a href="{% url 'web:garden-about' %}">What is Project GARDEN?</a>
         </p>
         <p>
-            <a href="#">New to Project GARDEN? Visit our How to Participate page!</a>
+            <a href="{% url 'web:garden-participate' %}">New to Project GARDEN? Visit our How to Participate page!</a>
         </p>
         <p>
-            <a href="#">Project GARDEN Studies</a>
+            <a href="{{ base_url }}{{ garden_url }}">Project GARDEN Studies</a>
         </p>
```

Comment:
Why does this need the `base_url`?

Reply:
It doesn't 🤦‍♀️  Thanks for catching this - fixed!